semantics/language/execution/check-loc.k

```diff
@@ -85,23 +85,23 @@ module C-CHECK-LOC
                (.Set => SetItem(basedOn(Base, Tag))) _)
           ...</k>
           <activeBlocks>... Tag |-> activity(_, Modified:Map) ...</activeBlocks>
-          requires Loc in keys(Modified)
+          requires Loc in_keys(Modified)
           [structural]
      rule <k> filterUnmodified(Loc:SymLoc, (basedOn(_, Tag:Scope) => .K), _, _)
           ...</k>
           <activeBlocks>... Tag |-> activity(_, Modified:Map) ...</activeBlocks>
-          requires notBool (Loc in keys(Modified))
```

Comment:
OCAML doesn't compute key sets in constant time, so I made a in_keys operator that will be in constant time.